virtio-devices/src/vsock/unix/muxer.rs

```diff
@@ -362,25 +362,24 @@ impl VsockMuxer {
 
     /// Handle/dispatch an epoll event to its listener.
     ///
-    fn handle_event(&mut self, fd: RawFd, evset: epoll::Events) {
+    fn handle_event(&mut self, fd: RawFd, event_set: epoll::Events) {
         debug!(
             "vsock: muxer processing event: fd={}, evset={:?}",
-            fd, evset
+            fd, event_set
```

Comment:
If you're going to change the variable name you should change the debug statement about that variable.

Reply:
Done